atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -535,24 +532,26 @@ static TimestampCache timestampCache(
             Optional<PersistentStore> timestampStore) {
```

Comment:
on second thought this is weird, a persistent store is a property of a particular timestamp cache, to get things setup correctly you have a weird ternary setup. If you're going to put the timestamp store through which requires manual wiring, then you might as well setup the timestamp cache properly.

Reply:
Reverted to original logic for setting up the cache.